crates/kitsune_p2p/dht/src/arq/strat.rs

```diff
@@ -99,15 +99,16 @@ pub struct ArqStrat {
 #[cfg(feature = "test_utils")]
 impl Default for ArqStrat {
     fn default() -> Self {
-        Self::standard(LocalStorageConfig::default())
+        Self::standard(LocalStorageConfig::default(), DEFAULT_MIN_PEERS as f64)
     }
 }
 
 impl ArqStrat {
     /// Standard arq strat
-    pub fn standard(local_storage: LocalStorageConfig) -> Self {
+    pub fn standard(local_storage: LocalStorageConfig, min_coverage: f64) -> Self {
```

Comment:
It wasn't actually possible to configure this with the tuning param... I haven't ended up needing this change yet on this PR but I do think the change needs making.